README.markdown

```diff
@@ -6669,6 +6673,78 @@ This feature was first introduced in the `v0.7.3` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.shared.DICT.capacity
+---------------------
```

Comment:
Better align `---` with the text in the previous line.

Reply:
done